libs/acn/PDU.cpp

```diff
@@ -69,6 +70,11 @@ bool PDU::Pack(uint8_t *buffer, unsigned int *length) const {
     offset += 1;
   }
 
+  if (m_force_length_flag) {
+    // TODO(Peter): Should this happen regardless of the force when we're
```

Comment:
also TO BE DONE before merging or some time after?

Reply:
Again I think one to review later. Partly because it would involve reading a load of spec docs, and partly because the answer may reveal itself either as I implement the remaining bits of code, or at something like the Plugfest testing with other systems.

Reply:
No, no special tests missing, it's just the keyword TODO that caught my eye

Reply:
I've added a note to check these in #1841